lessons/Early-Math/translating/langs/en-us/pages/practice-translating.adoc

```diff
@@ -0,0 +1,25 @@
+= Translating
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Each row represents a single arithmetic expression, written in three different forms. Fill in the empty spaces so that all three forms represent the same expression.
+
+[.FillVerticalSpace, cols="^.^1a,^.^4a,^.^4a,^.^4a", stripes="none", options="header"]
+|===
+| 	 | Expression in Words				| Circle of Evaluation		| Math Expression
+| @n | 24 increased by 3		 		| @show{(coe '(+ 24 3))}		| @ifsoln{@math{24 + 3}}
+| @n | @math{\frac {1}{3}} less than 4	| @showsoln{(coe '(- 4 1/3))}	| @math{4 - \frac {1}{3}}
+| @n | @ifsoln{the quotient of 24 and 8}| @showsoln{(coe '(/ 24 8))}	| @math{\frac {24}{8}}
```

Comment:
Probably better to use 24 \div 8 than \frac {24}{8}

Reply:
I actually think so important for teachers vary their representation of division, largely to help students feel more comfortable with fractions. Perhaps I should put a note indicating this? Not recognizing fractions as division can be a big barrier to student success in middle school math.

Reply:
Fifth grade CCSS standard - "Interpret a fraction as division of the numerator by the denominator (a/b = a ÷ b)."

Reply:
I totally agree that it's important for students to recognize fractions as a division. By using `/` instead of `\div` you are losing an opportunity to connect the two!

Reply:
I'm not using `/` here! I'm using `\frac` !